ui-components/src/page_components/sign-in/FormTerms.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useContext, useCallback, useMemo } from "react"
+import {
+  AuthContext,
+  AppearanceStyleType,
+  Button,
+  Field,
+  Form,
+  FormCard,
+  Icon,
+  MarkdownSection,
+  t,
+} from "@bloom-housing/ui-components"
+import Markdown from "markdown-to-jsx"
+import { useForm } from "react-hook-form"
+
+type FormTermsInValues = {
+  agree: boolean
+}
+
+const FormTerms = () => {
+  const { profile, userProfileService, loadProfile } = useContext(AuthContext)
+
+  // eslint-disable-next-line @typescript-eslint/unbound-method
+  const { handleSubmit, register, errors } = useForm<FormTermsInValues>()
+
+  const onSubmit = useCallback(async () => {
+    if (!profile) return
+
+    const jurisdictionIds =
+      profile?.jurisdictions.map((item) => ({
+        id: item.id,
+      })) || []
+
+    await userProfileService?.update({
+      body: { ...profile, jurisdictions: jurisdictionIds, agreedToTermsOfService: true },
+    })
+
+    loadProfile?.("/")
+  }, [loadProfile, profile, userProfileService])
+
+  const jurisdictionTerms = useMemo(() => {
+    return profile?.jurisdictions[0].partnerTerms
+  }, [profile])
+
+  return (
+    <>
+      <FormCard>
+        <div className="form-card__lead text-center">
```

Comment:
My preference would be to not have the text centered. I think it makes it harder to read.

![Screen Shot 2022-04-04 at 3 16 52 PM](https://user-images.githubusercontent.com/4401157/161641280-e829be9e-5173-46e8-974f-bf0760120862.png)

vs

![Screen Shot 2022-04-04 at 3 17 00 PM](https://user-images.githubusercontent.com/4401157/161641303-429f3838-65a4-434a-aa3a-b9d130c57fb3.png)